PowerUp/app/src/main/java/powerup/systers/com/FinalAvatarActivity.java

```diff
@@ -1,110 +1,104 @@
-        }
-
-        String skinImageName = getResources().getString(R.string.skin);
-        skinImageName = skinImageName + getmDbHandler().getAvatarSkin();
-        try {
-            photoNameField = ourRID.getClass().getField(skinImageName);
-            skinAvatar.setImageResource(photoNameField.getInt(ourRID));
-        } catch (NoSuchFieldException | IllegalAccessException
-                | IllegalArgumentException error) {
-            error.printStackTrace();
-        }
-
-        String clothImageName = getResources().getString(R.string.cloth);
-        clothImageName = clothImageName + getmDbHandler().getAvatarCloth();
-        try {
-            photoNameField = ourRID.getClass().getField(clothImageName);
-            clothAvatar.setImageResource(photoNameField.getInt(ourRID));
-        } catch (NoSuchFieldException | IllegalAccessException
-                | IllegalArgumentException error) {
-            error.printStackTrace();
-        }
-
-        String hairImageName = getResources().getString(R.string.hair);
-        hairImageName = hairImageName + getmDbHandler().getAvatarHair();
-        try {
-            photoNameField = ourRID.getClass().getField(hairImageName);
-            hairAvatar.setImageResource(photoNameField.getInt(ourRID));
-        } catch (NoSuchFieldException | IllegalAccessException
-                | IllegalArgumentException error) {
-            error.printStackTrace();
-        }
-        ImageView continueButton = (ImageView) findViewById(R.id.continueButtonFinal);
-        ImageView backButton = (ImageView) findViewById(R.id.backButtonFinal);
-        continueButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(FinalAvatarActivity.this);
-                boolean hasPreviouslyStarted = prefs.getBoolean(getString(R.string.preferences_has_previously_started), false);
-                if (!hasPreviouslyStarted) {
-                    SharedPreferences.Editor edit = prefs.edit();
-                    edit.putBoolean(getString(R.string.preferences_has_previously_started), Boolean.TRUE);
-                    edit.apply();
-                }
-                startActivityForResult(new Intent(FinalAvatarActivity.this, MapActivity.class), 0);
-            }
-        });
-        backButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                startActivity(new Intent(FinalAvatarActivity.this, AvatarRoomActivity.class));
-                overridePendingTransition(android.R.anim.fade_in, android.R.anim.fade_out);
-            }
-        });
-    }
-    public DatabaseHandler getmDbHandler() {
-        return mDbHandler;
-    }
-
-    public void setmDbHandler(DatabaseHandler mDbHandler) {
-        this.mDbHandler = mDbHandler;
-    }
-}
+package powerup.systers.com;
```

Comment:
Can you please check your editor settings, this should not happen.

Reply:
@m-murad I have checked the editor. Class is not auto aligned from the editor. I just tried to put original and check. Automatically its not import one class called "Random" .Y is that? so that my unchanged FinalAvatarActivity.java class have only 107 line from the first. 
According to the android studio warning I have done three changes to the class.
+ public Activity finalAvatarInstance to private Activity finalAvatarInstance
+ public DatabaseHandler getmDbHandler() to private DatabaseHandler getmDbHandler()
+ public void setmDbHandler(DatabaseHandler mDbHandler) to the private void setmDbHandler(DatabaseHandler mDbHandler)